endtoend/evaluators/validator.go

```diff
@@ -27,7 +27,7 @@ var ValidatorsAreActive = Evaluator{
 // ValidatorsParticipating ensures the expected amount of validators are active.
 var ValidatorsParticipating = Evaluator{
 	Name:       "validators_participating_epoch_%d",
-	Policy:     afterNthEpoch(1),
+	Policy:     afterNthEpoch(3),
```

Comment:
Given we only compute participation at epoch transition, not slot transition. The first time we see a 100% participation will be 2->3 transition

Reply:
Got it, what would participation be before the 3rd epoch?